src/lib.rs

```diff
@@ -1,31 +1,40 @@
 //! SputnikVM implementation, traits and structs.
 //!
-//! SputnikVM work on two different levels. It can directly handles a
-//! transaction, or a normal Etheruem execution context. To interact
-//! with the virtual machine, you usually only need to work with the
-//! [VM](trait.VM.html) trait.
+//! SputnikVM works on two different levels. It can directly handle
+//! either a transaction or a normal Ethereum execution context. To
```

Comment:
I'd suggest to emphasize this a little more, even. Maybe something like:

```
... on two different levels. It can directly handle either

1. a transaction, or
2. a normal Ethereum execution context.

To interact...
```